pytorch_lightning/metrics/functional/nlp.py

```diff
@@ -24,16 +25,20 @@ def _count_ngram(ngram_input_list: List[str], n_gram: int) -> Counter:
 
     for i in range(1, n_gram + 1):
         for j in range(len(ngram_input_list) - i + 1):
-            ngram_key = tuple(ngram_input_list[j : i + j])
+            ngram_key = tuple(ngram_input_list[j: i + j])
```

Comment:
@rohitgr7 [j:(i + j)]

Reply:
Thanks! My text editor made this update :)